### DISCUSSION TITLE
Key with nested json

### DISCUSSION THREAD

**Discussion by pchaudha** [shoelace-style/localize#24] [General]
Is there any way we can use nested locale key? e.g.

```js
{
  abc: {
    xyz: "XYZ"
  }
}
```

 so can we use something like "abc.xyz" to get translated value as "XYZ"

**Comment by claviska** [comment]
No, nested keys aren't supported and I don't think it's something we want to add because then we'd need to parse keys and deal with types, which will likely cause a lot of bloat for little benefit.

You can, however, use whatever you want for the key — they just need to be defined top-level.

**Reply by pchaudha** [reply]
Okay, Thanks
